### PR TITLE
Add PIT marker for Rocket team components

### PR DESCRIPTION
**Pull request by shubhamsg199**
Adding PIT marker to Rocket team components

### CODE REVIEW

**Review by jameerpathan111 — APPROVED** [review]

**Review by amolpati30 — APPROVED** [review]

**Review by ogajduse — APPROVED** [review]

**Comment by ogajduse** [comment]
@shubhamsg199 Please consider adding labels for the older versions so the tests get picked in 6.13 and 6.14 testing.

**Review by ColeHiggins2 — APPROVED** [review]

File: tests/foreman/cli/test_registration.py [hunk 1]
```diff
@@ -190,6 +190,7 @@ def test_negative_register_twice(module_ak_with_cv, module_org, rhel_contenthost
     assert 'This system is already registered' in str(result.stderr)
 
 
+@pytest.mark.pit_client
```
Comment (shweta83) [inline tests/foreman/cli/test_registration.py:193]: we are testing force registration in "test_global_registration_form_populate" so we can skip this one and we can test "test_positive_allow_reregistration_when_dmi_uuid_changed" instead for pit.
Wdyt @ogajduse ?

**Comment by shweta83** [comment]
@shubhamsg199  We should test pit for at least one compute resource azure.

**Review by shweta83 — APPROVED** [review]
Ack

File: tests/foreman/cli/test_registration.py (hunk 1 above)
Comment (shubhamsg199) [reply tests/foreman/cli/test_registration.py]: Agreed, marked `test_positive_allow_reregistration_when_dmi_uuid_changed` for pit

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**